workers/unity/Packages/com.improbable.gdk.playerlifecycle/Config/PlayerLifecycleConfig.cs

```diff
@@ -11,22 +11,19 @@ public static class PlayerLifecycleConfig
         /// <summary>
         ///     The time in seconds between player heartbeat requests.
         /// </summary>
-        public const float PlayerHeartbeatIntervalSeconds = 5f;
+        public static float PlayerHeartbeatIntervalSeconds = 5f;
```

Comment:
Why `const` -> `static` ?

Reply:
This is to allow users to change these variables in-code, outside of this class